lib/src/git.rs

```diff
@@ -1273,10 +1276,13 @@ fn create_git_ref(
     git_ref_name: &GitRefName,
     new_oid: gix::ObjectId,
 ) -> Result<(), FailedRefExportReason> {
-    if let Ok(git_repo_ref) = git_repo.find_reference(git_ref_name.as_str()) {
+    if let Some(git_ref) = git_repo
```

Comment:
> I also renamed git_repo_ref to git_ref because there aren't any other "git_ref" variables.

I suspect the idea was to distinguish it from `View::git_refs`, but there's no view object in scope here, so that's not needed. (no action requested)